sechub-pds-solutions/scancode/10-create-image.sh

```diff
@@ -49,6 +49,16 @@ if [[ ! -z "$SCANCODE_VERSION" ]] ; then
     BUILD_ARGS+=" --build-arg SCANCODE_VERSION=$SCANCODE_VERSION"
 fi
 
+if [[ ! -z "$SPDX_TOOL_VERSION" ]] ; then
```

Comment:
If `SPDX_TOOL_VERSION` and `SPDX_TOOL_CHECKSUM` are unset, the Docker image build fails now.

Also blanks inside a Docker build argument (SPDX_TOOL_CHECKSUM) did not work for me:
```
> export SPDX_TOOL_VERSION="1.1.5"
> export SPDX_TOOL_CHECKSUM="feff2d5e7f66576a6e411d65a18046dbaf7ed6bad6e223a1c0ecee075a72b70b  tools-java-1.1.5-jar-with-dependencies.jar"
> export SCANCODE_VERSION=31.2.4
> ./10-create-image.sh my.registry/sechub/pds-scancode-test 0.999.0 my.registry/sechub/pds_base_image:latest
>> Base image: my.registry/sechub/pds_base_image:latest
>> Scancode version: 31.2.4
>> SPDX Tool version: 1.1.5
>> SPDX Tool checksum: feff2d5e7f66576a6e411d65a18046dbaf7ed6bad6e223a1c0ecee075a72b70b  tools-java-1.1.5-jar-with-dependencies.jar
"docker build" requires exactly 1 argument.
See 'docker build --help'.
...
```

As a solution, we could set `SPDX_TOOL_CHECKSUM="feff2d5e7f66576a6e411d65a18046dbaf7ed6bad6e223a1c0ecee075a72b70b"`
and the check in the Dockerfile (replace lines 52-54) like this:
```Dockerfile
echo "${SPDX_TOOL_CHECKSUM} tools-java-${SPDX_TOOL_VERSION}-jar-with-dependencies.jar" | sha256sum -c
```